site/en/blog/sxg-desktop/index.md

```diff
@@ -0,0 +1,103 @@
+- Support for server side personalization 
+
+Read on to learn more about these features and how you can leverage them to improve the user experience on your site.
+
+## Support for Desktop Navigations
+
+With this release, Signed Exchanges will now be available for Desktop cross-origin navigations on Chromium browsers. This builds on the existing support for Android mobile and means that now, you can use SXG to also cache and prefetch the desktop version of your site. 
+
+We estimate<sup><a href="#footnote-1">1</a></sup> that this release will, on average, double SXG's coverage of your site's visits, enabling improved loading and performance for more users. Today on mobile, this speed improvement is between 300 and 400 ms of LCP reduction on average for navigations from Google Search.
+
+### How can you leverage this feature
+
+Enabling Signed Exchanges for the Desktop version of your site does not require any additional work on your part. This feature is already supported on both Cloudflare's easy-to-use [Automatic Signed Exchanges tool](https://support.cloudflare.com/hc/articles/4411075595661-Automatic-Signed-Exchanges-SXGs-Beta) and the [open source SXG tool-kit](https://web.dev/signed-exchanges/#tooling).
+
+<div class="table-wrapper scrollbar">
+  <table></table>
+    <tr>
+      <th>
+        <strong>Existing SXG Mobile Adopters:</strong>
+      </th>
+      <td>
+        SXG will automatically begin applying to your desktop navigations.
+      </td>
+    </tr>
+    <tr>
+      <th><strong>New Adopters:</strong></th>
+      <td>
+        When enabled, SXG is available for both Desktop and Android Mobile navigations, by default. Visit the SXG <a href="https://web.dev/signed-exchanges/#tooling" rel="noopener">blog post</a> to learn how you can enable it for your site.</td>
+    </tr>
+  </table>
+</div>
+
+You can opt out of this if you want using the supported-media meta tag. Learn more [here](https://github.com/google/webpackager/blob/main/docs/supported_media.md).
+
+{% Aside %}
+**Learn more about the impact of Signed Exchanges**
+
+Several global brands and sites have already benefited from Signed Exchanges. Let's look at a few case studies for current SXG adopters on Android mobile:
+
+Cloudflare, a leading Content Distribution Network, ran an experiment to measure the impact of Signed Exchanges for its customers of the 500 sites they tested:
+
+- **85%** saw an **improvement in LCP**.
+- **98%** observed reduced **Time to First Byte (TTFB)**.
+- Median **20% improvement across these metrics**.
+
+[RebelMouse](https://www.rebelmouse.com/signed-exchange?utm_medium=post&utm_source=google&utm_campaign=signed-exchange-co-marketing&utm_content=signed-exchange), a prominent Content Management System, recorded an improvement in business and performance metrics for their customers including:
+
+- **41% LCP improvement** for Narcity.
+- **27% increase in sessions per user** for Paper Magazine.
+- **21% decrease in loading time** for MLT Blog.
```

Comment:
Arbitrary capitalization is strongly discouraged in the style guide, reconsider the formatting here.